templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -499,13 +508,26 @@ contents:
 
       extra_bridge_file='/etc/ovnk/extra_bridge'
 
-      # Some deployments uses a temporary solution where br-ex is moved out from the default gateway interface
-      # and bound to a different nic. If that is the case, we rollback (https://github.com/trozet/openshift-ovn-migration).
-      if [ "$iface" != "br-ex" ] && [  -f "$extra_bridge_file" ] && nmcli connection show br-ex &> /dev/null; then
-        echo "Default gateway is not bridge but bridge exists, reverting"
-        remove_all_ovn_bridges
-        reload_nm
+      if [ ! -f /run/configure-ovs-boot-done ]; then
```

Comment:
why is this moved? don't we need to rollback before we detect the default gateway interface?

Reply:
ups that's true, I will move it back where it was.

Reply:
we probably need an ovn upgrade job here of some kind